packages/graphql/src/modules/user/utils/index.ts

```diff
@@ -0,0 +1,13 @@
+import User from '../domain/User';
+import {UserGraphQLType} from '../presentation/UserType';
+
+export const transformDomainUserIntoPresentationUser = (user: User.Type): UserGraphQLType => {
+  const lastLoggedInProvider = user.providers.sort(provider => provider.lastLogIn)[0];
+
+  return {
+    id: user.id,
+    name: lastLoggedInProvider.name,
+    email: lastLoggedInProvider.email,
+    profileImage: lastLoggedInProvider.profileImage,
+  };
+};
```

Comment:
Why is this function located within a `index` file within an empty folder (`utils`)? 

Wouldn't make more sense to name it `utils.ts` and place it within `packages/graphql/src/modules/user`?